ci-operator/step-registry/ipi/conf/aws/min-perms-user/ipi-conf-aws-min-perms-user-commands.sh

```diff
@@ -0,0 +1,131 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+trap 'CHILDREN=$(jobs -p); if test -n "${CHILDREN}"; then kill ${CHILDREN} && wait; fi' TERM
+trap 'rm -rf /tmp/aws_cred_output /tmp/pull-secret /tmp/min_perms/' EXIT TERM INT
+
+function run_command() {
+    local cmd="$1"
+    echo "Running Command: ${cmd}"
+    eval "${cmd}"
+}
+
+function installer_generate_policies()
+{
+	local dir=/tmp/min_perms/
+
+	mkdir -p ${dir}
+
+	# Make a copy of the install-config.yaml since the installer will consume it.
```

Comment:
> installer will consume it

`create manifests` and `create ignition-configs` consume config file as well, what's the correct order after `permissions-policy` is added? is `permissions-policy` -> `manifests` -> `ignition-configs` ?

Reply:
As long as the workdir is preserved, you can run `create permissions-policy` at any point after `create install-config` since the installer will use either the `install-config.yaml` file from disk or from the state file `<workdir>/.openshift_install_state.json`. The `permissions-policy` commands generates the same set of permissions that is used during install-config validation.

Reply:
The ignition configs depend on the manifests (you could edit the manifests generated by the installer), so `manifests -> ignition-configs` is the correct order.